client/src/main/java/com/battleships/models/board/Coordinate.java

```diff
@@ -18,7 +37,7 @@ public boolean equals(Object o) {
             return false;
         } else {
             Coordinate that = (Coordinate)o;
-            return this.row == that.row && this.col == that.col;
+            return this.row == that.row && this.column == that.column;
```

Comment:
return (this.row == that.row && this.column == that.column); 
would be more readable

Reply:
fixed